Dockerfile

```
@@ -2,4 +2,7 @@ FROM golang:1.14-alpine
 
 COPY mockery /
 
+# allow mockery to be accessible from $PATH
+RUN ln -s /mockery /usr/local/bin
+
 ENTRYPOINT ["/mockery"]
```

Something akin to this?

```suggestion
COPY mockery /usr/local/bin

ENTRYPOINT ["/usr/local/bin/mockery"]
```